site/docs/integrations/astro.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Astro
+parent: integrations
+---
+
+# Astro
+
+A plugin for integrating vanilla-extract with [Astro](https://astro.build).
```

Comment:
This makes it sound like there's a separate astro plugin, but that's not the case. Just reworded it a bit.

```suggestion
Vanilla Extract integrates with [Astro](https://astro.build) via the Vite plugin, rather than a separate Astro plugin.
```